content/_index.md

```diff
@@ -1,8 +1,78 @@
 +++
-title = "Snowplow Accelerator"
-menuTitle="Introduction"
+title = "Introduction"
+menuTitle = "Introduction"
+pre = "<i class='fas fa-rocket'></i> "
 chapter = false
 weight = 1
 +++
 
-This is an example of an Accelerator Introduction
\ No newline at end of file
+### Web Performance Accelerator
+
+#### Introduction
+
+This accelerator will show you how to track and model raw `core web vital` events so that you can monitor your website's performance via the most essential metrics. Improving the web performance can have a dramatic impact on how your users view and interact with your website with the potential to maximize your revenue.
+
+!['Potential impact'](images/impact.png?height=30pc)
+
+Here you will learn to:
```

Comment:
Let's add a screenshot of some of the Tableau work at the start of this so users can get an idea of what they can expect to see / generate by following the process.